poetry.lock

```diff
@@ -1127,6 +1140,10 @@ azure-mgmt-storage = [
     {file = "azure-mgmt-storage-17.0.0.zip", hash = "sha256:c0e3fd99028d98c80dddabe1c22dfeb3d694e5c1393c6de80766eb240739e4bc"},
     {file = "azure_mgmt_storage-17.0.0-py2.py3-none-any.whl", hash = "sha256:671a2c57bff42e32e18ea7891d2b81d6dcdbb454fe60dc7be7ab09b92c11fd01"},
 ]
+azure-storage-blob = [
+    {file = "azure-storage-blob-12.8.0.zip", hash = "sha256:36b85a3423379d4a93f663022487cf53aa3043a355f8414321dde878c00cb577"},
```

Comment:
where is this file coming from originally?

Reply:
This was autogenerated after I added `azure-storage-blob` to the `pyproject.toml` file.

Reply:
@juhlee-microsoft Since I am new to this, please let me know if this needs to be done differently.

Reply:
It's not clear where this file is coming and where it is stored. You will need to make sure of it before merging. Raise the red flag here.

Reply:
Hi Ju, this file is auto generated, and we don't modify its content manually. The file is used to make sure every place installed the same versions of dependencies.